config/routes.rb

```diff
@@ -419,12 +419,12 @@
   get 'status', :to => 'home#status', :as => "status"
 
   # get only for alterator unattended scripts
-  get 'unattended/provision/:metadata', :controller => 'unattended', :action => 'host_template', :format => 'html',
+  get 'unattended/provision/:metadata', :controller => 'unattended', :action => 'host_template', :format => 'text',
     :constraints => { :metadata => /(autoinstall\.scm|vm-profile\.scm|pkg-groups\.tar)/ }
   # get for end of build action
-  get 'unattended/built/(:id(:format))', :controller => 'unattended', :action => 'built'
+  get 'unattended/built/(:id(:format))', :controller => 'unattended', :action => 'built', :format => 'text'
   # get for all unattended scripts
-  get 'unattended/(:kind/(:id(:format)))', :controller => 'unattended', :action => 'host_template'
+  get 'unattended/(:kind/(:id(:format)))', :controller => 'unattended', :action => 'host_template', :format => 'text'
```

Comment:
Does not seem to work:
```
1) Failure:
UnattendedControllerTest#test_0008_unattended files content type should be text/plain [test/functional/unattended_controller_test.rb:89]:
Failed assertion, no message given.
```

Even tried `defaults: {:format => 'text'}` without any luck. I don't want to mess around, keeping the `set_content_type` there.